### DISCUSSION TITLE
filter by upvotes/reactions

### DISCUSSION THREAD

**Discussion by Pacerier** [community/community#60953] [Projects and Issues]
I need a list of all upvotes/reactions (issues) on GH of a particular user (eg myself).

Where is the list?

How to filter it? (eg by repo)

---
<sup>&bull; [misc](https://github.com/orgs/community/discussions/60954)</sup>

**Comment by Pacerier** [comment]
[backlink #11935]

**Comment by studgeek** [comment]
Not exactly the same, but you can view issues you are subscribed to at https://github.com/notifications/subscriptions.
